src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -261,29 +260,63 @@ metadata:
+__Note__: When applying the example in the snippet above, please replace the `CLUSTER_NAME` placeholder with the name of the workload cluster which you are configuring and the `CLUSTER_NAMESPACE_NAME` placeholder with the name of the namespace which contains the cluster.
 
 Then submit the resource to the management API and the App operator will manage it to make the actual installation and configuration. You can log in now into the cluster API with your identity provider using the login endpoint that Dex creates for you. By default, it will be `https://login.CLUSTER_NAME.BASE_DOMAIN`.
 
-__Warning__: It is assumed that you have an [ingress controller and cert-manager]({{< relref "/getting-started/app-platform/deploy-app" >}}) running in your cluster in order to make dex available for the callback request made by your identity provider securely. If you supply custom certificates when deploying dex, then you can skip cert-manager installation. Both of these apps are offered in our managed app catalog.
+__Warning__: It is assumed that you have an [ingress controller and cert-manager]({{< relref "/getting-started/app-platform/deploy-app" >}}) running in your cluster in order to make Dex available for the callback request made by your identity provider securely. Both of these apps are offered in our managed app catalog. If you supply custom certificates when deploying Dex, then you can skip cert-manager installation. 
+
+### Deploying Dex with custom certificates
+
+Dex gets certificates from the cert-manager by default. In case the cert-manager is configured to use a custom certification authority, which is not publicly trusted, Dex needs to be configured to trust it. The custom certification authority needs to be stored in a `Secret` and referenced in the `trustedRootCA` section of the `Secret` or `ConfigMap` with configuration values for the Dex app.
+
+```yaml
+trustedRootCA:
+  secretName: ... # Name of the secret, which contains the custom CA
+  name: ... # Name of the property inside the secret
+```
+
+It is also possible to disable retrieving certificates from the cert-manager and supply custom certificates to Dex directly. They can be configured in the `ingress` section of the `Secret` or `ConfigMap` with the configuration values for the Dex app:
+
+```yaml
+ingress:
+  tls:
+    letsencrypt: false
+    caPemB64: ...
+    crtPemB64: ...
+    keyPemB64: ...
+```
+
+### Running Dex in a private workload cluster
+
+When deploying Dex to a private workload cluster, it is required to specify a proxy configuration in order to ensure that Dex has access to the outside network. Proxy setup can be provided in the `proxy` section of the `Secret` or `ConfigMap` with configuration values for the Dex app. It is required to set the proxy as `enabled`, specify the URL of the HTTPS proxy in the `https_proxy` property and exclude Kubernetes API from running through the proxy by adding its IP address or network to the `no_proxy` property.  
```

Comment:
Consider making new lines after sentences. When someone makes changes later, the PRs are tidier that way.